File: simple_pairs.py
def pairs(arr):
    if len(arr) % 2 != 0:
        arr.pop(-1)
    count = 0
    while len(arr) > 0:
        first = arr.pop(0)
        second = arr.pop(0)
        biggest = max(first, second)
        smallest = min(first, second)
        if biggest - smallest == 1:
            count += 1
    return count

File: simplePairs.js
function pairs(ar){
    let count = 0;
    if (ar.length % 2 != 0){
      ar.pop();
    }
     while (ar.length > 0){
       let first = ar.shift();
       let second = ar.shift();
       let biggest = Math.max(first, second);
       let smallest = Math.min(first, second);
       if (biggest - smallest == 1){
         count ++;
       }
     }
     return count;
   };